pages/express-relay/integrate-as-protocol.mdx

```diff
@@ -0,0 +1,222 @@
+The SDK exposes [`IExpressRelay`](https://github.com/pyth-network/pyth-crosschain/blob/main/express_relay/sdk/solidity/IExpressRelay.sol) and [`IExpressRelayFeeReceiver`](https://github.com/pyth-network/pyth-crosschain/blob/main/express_relay/sdk/solidity/IExpressRelayFeeReceiver.sol) interfaces to interact with Express Relay.
+
+<Tabs items={['Hardhat', 'Foundry']}>
+  <Tabs.Tab>
+  If you are using Hardhat, you can install the SDK using npm:
+
+```bash copy
+npm install @pythnetwork/express-relay-sdk-solidity
+```
+  </Tabs.Tab>
+  <Tabs.Tab>
+  If you are using Foundry, you must create an NPM project if you don't already have one. From the root directory of your project, run:
+
+```bash copy
+npm init -y
+npm install @pythnetwork/express-relay-sdk-solidity
+```
+
+Then add the following line to `remappings.txt` file:
+
+```text copy
+@pythnetwork/express-relay-sdk-solidity/=node_modules/@pythnetwork/express-relay-sdk-solidity
+```
+  </Tabs.Tab>
+</Tabs>
+
+### Modifying the Protocol's Contract
+
+The following steps show how to modify your protocol's contract to verify if the current transaction is permissioned by Express Relay and to receive the auction proceedings.
+
+1. Call the [`isPermissioned`](https://github.com/pyth-network/pyth-crosschain/blob/main/express_relay/sdk/solidity/IExpressRelay.sol#L10C14-L10C28) method from `IExpressRelay` interface to make sure the current transaction is permissioned by Express Relay.
+1. Implement the [`IExpressRelayFeeReceiver`](https://github.com/pyth-network/pyth-crosschain/blob/main/express_relay/sdk/solidity/IExpressRelayFeeReceiver.sol#L4) interface to **receive** auction proceedings. 
+
+#### 1. Verify Permissioning
+
+The `isPermissioned` function takes two arguments:
+1. `protocolFeeReceiver`: The address to receive the protocol's share of auction proceedings.
+1. `permissionId`: A unique identifier for the opportunity.
+
+```solidity copy
+import "@pythnetwork/express-relay-sdk-solidity/IExpressRelay.sol";
+
+// Express Relay contract address on Optimism Sepolia
+// 
+// Check https://docs.pyth.network/express-relay/contract-addresses 
+// for the address deployed on other networks
+address expressRelay = 0xD6e417287b875A3932c1Ff5dcB26D4D2C8b90B40;
+
+require(
+		IExpressRelay(expressRelay).isPermissioned(
+        protocolFeeReceiver,
+        permissionId
+    ),
+		"not permissioned"
+);
+```
+<Callout type="info" emoji="ℹ️">
+  The `permissionId` represents unique identifying information of an opportunity. 
+  For a liquidation opportunity, the vault address or ID could be concatenated into `bytes` format. 
+  Consult [`Permissioning`](./how-express-relay-works/permissioning.mdx) for more information on generating permission IDs. 
+</Callout>
+
+#### 2. Set up Fee Receiver
+Express Relay will call the `receiveAuctionProceedings` method present in `IExpressRelayFeeReceiver`. The call will transfer the protocol's share of the auction proceeding to the `protocolFeeReceiver` address.
```

Comment:
Why do you feel to put the reasoning before the action?

Reply:
BTW, I am okay with both.

Reply:
I think the first sentence is not informative. But I'm okay with both versions
Just thought it would be better to communicate it through a single sentence. I'm good with what you prefer here.